scss/_dropdown.scss

```diff
@@ -24,6 +24,15 @@
   }
 }
 
+.dropup {
+  .dropdown-toggle {
+    &:after {
```

Comment:
Fixed. Sorry about that. I wasn't sure if you wanted double or single as I hadn't heard about #17058 yet.